templates/apps/sumo-aws-apps/firewall-manager-org/firewall-manager-org-waf-network-firewall-policy.yaml

```diff
@@ -138,9 +138,7 @@ Resources:
           ErrorOutputPrefix: "SumoLogic-Kinesis-Failed-Logs/"
           RoleARN: !GetAtt KinesisRole.Arn
 
-  rFmsRegionalWafDefaultPolicy:
```

Comment:
these policies needs KinesisFirehoseDeliveryStreamArn so it should exists before we create these policies

Reply:
if Create Delivery Stream then the LoggingConfiguration will need KinesisFirehoseDeliveryStreamArn. In case Create Delivery Stream is not used, LoggingConfiguration will not be configured

Reply:
`Dependent stacks also have implicit dependencies in the form of target properties !Ref, !GetAtt, and !Sub`. https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-dependson.html. 
 so we may not need explicity depends on hence resolving.